### PR TITLE
chore: create dirs and README template for all ee bridges

### PR DESCRIPTION
**Pull request by HJianBo**
Fixes https://emqx.atlassian.net/browse/EMQX-9574

### <samp>🤖 Generated by Copilot at 3587d19</samp>

This pull request adds BSL license files and README.md files to 10 EMQX bridge applications that are part of the EMQX Enterprise Edition. The BSL license files grant limited rights to use the bridge applications for non-production purposes, while the README.md files provide brief introductions and documentations for the bridge applications. The bridge applications allow users to connect EMQX and various data sources and sinks and ingest IoT data using the EMQX rules engine.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- ~Added tests for the changes~
- ~Changed lines covered in coverage report~
- ~Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files~
- [x] For internal contributor: there is a jira ticket to track this change
- ~If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up~
- ~Schema changes are backward compatible~

## Checklist for CI (.github/workflows) changes

- ~If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)~
- ~Change log has been added to `changes/` dir for user-facing artifacts update~

### CODE REVIEW

File: apps/emqx_bridge_gcp_pubsub/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# EMQX GCP Pub/Sub Bridge
+
+[Google Cloud Pub/Sub](https://cloud.google.com/pubsub) is a messaging service provided by Google Cloud Platform (GCP).
+
+The application is used to connect EMQX and GCP Pub/Sub.
+User can create a rule and easily ingest IoT data into GCP Pub/Sub by leveraging
+the EMQX [EMQX Rules](https://docs.emqx.com/en/enterprise/v5.0/data-integration/rules.html).
```
Comment (thalesmg) [inline apps/emqx_bridge_gcp_pubsub/README.md:7]:
```suggestion
the [EMQX Rules](https://docs.emqx.com/en/enterprise/v5.0/data-integration/rules.html).
```
(similar in other places)

File: apps/emqx_bridge_rocketmq/src/emqx_bridge_rocketmq.app.src [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{application, emqx_bridge_rocketmq, [
+    {description, "EMQX Enterprise RcoketMQ Bridge"},
```
Comment (thalesmg) [inline apps/emqx_bridge_rocketmq/src/emqx_bridge_rocketmq.app.src:2]:
```suggestion
    {description, "EMQX Enterprise RocketMQ Bridge"},
```

File: apps/emqx_bridge_cassandra/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+with no single point of failure.
+It is commonly used in web and mobile applications, IoT, and other systems that
+require storing, querying, and analyzing large amounts of data.
+
+The application is used to connect EMQX and Cassandra. User can create a rule
+and easily ingest IoT data into Cassandra by leveraging the
+[EMQX Rules](https://docs.emqx.com/en/enterprise/v5.0/data-integration/rules.html).
+
+
+# Documention
```
Comment (thalesmg) [inline apps/emqx_bridge_cassandra/README.md:15]:
```suggestion
# Documentation
```
(similar occurrences in other files)

File: apps/emqx_bridge_timescale/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+providing automatic partitioning across time and space (partitioning key), as well as full SQL support.
+
+The application is used to connect EMQX and TimescaleDB.
+User can create a rule and easily ingest IoT data into TimescaleDB by leveraging
+the EMQX [EMQX Rules](https://docs.emqx.com/en/enterprise/v5.0/data-integration/rules.html).
+
+
+# Documention
+
+- Refer to [Ingest data into TimescaleDB](todo)
```
Comment (thalesmg) [inline apps/emqx_bridge_timescale/README.md:15]: Maybe leave a placeholder URL that follows the pattern for other bridges?

```suggestion
- Refer to [Ingest data into TimescaleDB](https://docs.emqx.com/en/enterprise/v5.0/data-integration/data-bridge-timescaledb.html)
```

**Review by jinfahua — DISMISSED** [review]
Fix the comments from Thales, others are LGTM.

File: apps/emqx_bridge_clickhouse/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# EMQX ClickHouse Bridge
+
+[ClickHouse](https://github.com/ClickHouse/ClickHouse)is an open-source, column-based
```
Comment (jinfahua) [inline apps/emqx_bridge_clickhouse/README.md:3]: Add a blank before `is`.

File: apps/emqx_bridge_cassandra/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+[Apache Cassandra](https://github.com/apache/cassandra) is an open-source, distributed
+NoSQL database management system that is designed to manage large amounts of structured
+and semi-structured data across many commodity servers, providing high availability
+with no single point of failure.
+It is commonly used in web and mobile applications, IoT, and other systems that
+require storing, querying, and analyzing large amounts of data.
+
+The application is used to connect EMQX and Cassandra. User can create a rule
+and easily ingest IoT data into Cassandra by leveraging the
```
Comment (jinfahua) [inline apps/emqx_bridge_cassandra/README.md:11]: `leveraging the` > remove `the`

File: apps/emqx_bridge_hstreamdb/README.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# EMQX HStreamDB Bridge
+
+[HStreamDB](https://hstream.io/)is streaming database purpose-built to ingest,
```
Comment (jinfahua) [inline apps/emqx_bridge_hstreamdb/README.md:3]: add blank before `is`.

**Review by jinfahua — CHANGES_REQUESTED** [review]
Several minor changes.

**Review by jinfahua — APPROVED** [review]

**Review by zmstone — APPROVED** [review]

**Pull request merged**